src/installer/managed/Microsoft.NET.HostModel/Microsoft.NET.HostModel.csproj

```diff
@@ -19,7 +19,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Reflection.Metadata" Version="1.8.0" />
+    <PackageReference Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataVersion)" />
```

Comment:
We need to ensure that we're matching the version of `System.Reflection.Metadata` that is in the SDK, as version mismatches can cause significant issues when running on the Full Framework version of MSBuild.

Reply:
Looks like we can update to 6.0.0 (the current value of `SystemReflectionMetadataVersion`) as it's the same as the sdk's version.

Reply:
Yup, I first tested updating `SystemReflectionMetadataVersion` to 7.0 preview7, but `ILCompiler.Build.Tasks.csproj` requirement is that we keep `SystemCollectionsImmutableVersion` and `SystemReflectionMetadataVersion` in sync. Updating `System.Collections.Immutable` to 7.0 preview7  was failing Pri0 tests (`XUnitWrapperGenerator` was unable to load S.C.I among other things), so I reverted that change.